workitem/link/link_repository.go

```diff
@@ -111,6 +113,10 @@ func (r *GormWorkItemLinkRepository) Create(ctx context.Context, sourceID, targe
 		}
 		return nil, errors.NewInternalError(db.Error.Error())
 	}
+	// save a revision of the created work item link
+	if err := r.revisionRepo.Create(ctx, creatorID, RevisionTypeCreate, *link); err != nil {
+		return nil, errs.WithStack(err)
```

Comment:
@xcoulon That call doesn't really wrap the Error (think Cause in Java), it only adds the error message to the new error message.